dagshub/streaming/filesystem.py

```diff
@@ -481,10 +481,11 @@ def scandir(self, path='.'):
     @cache_by_path
     def _api_listdir(self, path: str, include_size: bool = False):
         return requests.get(f'{self.content_api_url}/{path}', auth=self.auth,
-                            params={'include_size': 'true'} if include_size else {})
+                            params={'include_size': 'true'} if include_size else {},
+                            headers=config.requests_headers)
 
     def _api_download_file_git(self, path: str):
-        return requests.get(f'{self.raw_api_url}/{path}', auth=self.auth)
+        return requests.get(f'{self.raw_api_url}/{path}', auth=self.auth, headers=config.requests_headers)
```

Comment:
You're sure that setting `headers=` won't cause other headers that SHOULD have been sent by default, to be omitted?

Reply:
Yes, I checked that, they have a merge function of default + user requested.